.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## This pull request makes the following changes
+
+Fixes ushahidi/platform-client-mzima#replace-this-text-with-issue-ticket-number
+
+## Pull request change details
+
+What fixes does this pull request contain? - Add them here using ordered or unordered list.
+
+## Testing checklist
+
+What reviewers or QA should check for while reviewing this pull request:
+
+[ ] Add checklist item 1 here
+[ ] Add cheklist item 2 here... Continue adding as much checklist items as you would like
+[ ] I certify that I ran my checklist
```

Comment:
This is what your submission looks like in the screenshot below. You can see that the checkboxes are not well formatted @shakirandagire 

#

![Screenshot 2023-10-23 at 12 08 56](https://github.com/ushahidi/platform-client-mzima/assets/45185388/a648f2ca-e826-4ef0-b11e-7824e0d05be3)

Reply:
Make reference my previous comment and also fix your formatting for unordered list: https://github.com/ushahidi/platform-client-mzima/pull/657#discussion_r1368254806

In summary, use the exact formatting in that comment for your submission @shakirandagire

Reply:
@Ifycode I had primarily picked the text from the comment and not the formatting. On it now

Reply:
@Ifycode I have rectified the formatting of the PR template and it is currently like below
<img width="1310" alt="Screenshot 2023-10-23 at 14 39 51" src="https://github.com/ushahidi/platform-client-mzima/assets/6957960/747cc6f7-c87e-4efa-bbc5-7b57e86efaa9">